.github/workflows/rust-ci.yml

```
@@ -0,0 +1,89 @@
+name: Rust CI
+
+on:
+  push:
+    branches: [ "main", "master" ]
```

We don't have a master branch, only main
```suggestion
    branches: [ "main" ]
```